packages/api/package.json

```diff
@@ -13,17 +13,23 @@
     "test": "jest",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
     "test:e2e": "jest --config ./test/jest-e2e.json",
-    "generate": "prisma generate"
+    "generate": "prisma db push"
   },
   "dependencies": {
     "@coderscamp/shared": "0.0.1",
     "@hapi/joi": "17.1.1",
     "@nestjs/common": "7.6.18",
+    "@nestjs/config": "0.6.3",
```

Comment:
There is probably no need for the config module as we have our `env.ts` file to validate environment variables and make them type-safe